jf_agent/git/__init__.py

```diff
@@ -506,8 +516,7 @@ def get_nested_repos_from_git(git_connection, config: GitConfig):
             ]
             output_dict[org] = [x.name for x in org_repos]
 
-    elif config.git_provider == 'gitlab':
-
+    elif config.git_provider in ['gitlab', 'gitlab_v3']:
```

Comment:
The local executions of the agent with the `-m validate` flag look good to me, although I did notice that the repository and project name are being used interchangeably in the console output. I assumed this was a feature since the code seems to treat projects like repos but another set of eyes couldn't hurt. Here is the console output:

`dayo@ip-192-168-247-39:jf_agent(OJ-11509-gitlabv3-adapter)$ docker run --rm \                                                     
--mount type=bind,source=/Users/dayo/code_base/jf_agent/config.yml,target=/home/jf_agent/config.yml \
--mount type=bind,source=/Users/dayo/code_base/jf_agent/output,target=/home/jf_agent/output \
--mount type=bind,source=/Users/dayo/code_base/jf_agent,target=/home/jf_agent \
--env-file ./creds.env \
jellyfishco/jf_agent:latest -m validate --jellyfish-api-base http://docker.for.mac.localhost:8000
WARNING: Disabling SSL certificate validation
Validating configuration...

No Jira URL or credentials provided, skipping Jira validation...

Git details for instance 1/1:
  Provider: gitlab_v3
  Included projects: [2]
  Included repos: []
==> Testing Git connection...
✓
downloading gitlab projects... ✓
downloading repos for finalProject:   0%|          | 0/2 [00:00<?, ?repos/s]
downloading repos for finalProject:  50%|█████     | 1/2 [00:01<00:01,  1.75s/repos]
downloading repos for finalProject: 100%|██████████| 2/2 [00:04<00:00,  2.01s/repos]✓
  All projects and repositories available to agent:
  -- finalProject
    -- 3
    -- 2

Memory & Disk Usage:
  Available memory: 4235.23 MB
  Disk usage for jf_agent/output: 136 GB / 466 GB
  Size of jf_agent/output dir: 221M

Done`